### ISSUE TITLE
Using vue-ray, or js-ray etc - In browser Ray is being blocked by CORS policy

### ISSUE THREAD

**Issue by ijpatricio** [buggregator/app#72]
CORS allowed headers are missing.

Still didn't dive into source code. Is this something you would accept a PR?

Or you see this is missing, and easy fix on your end?

![image](https://user-images.githubusercontent.com/26031459/154250945-72fcc531-b1c4-4e2b-a92f-ce8c450b4d7d.png)

**Comment by butschster** [comment]
Hi @ijpatricio.

> Is this something you would accept a PR?
Sure

Try to check CORS on RoadRunner side https://github.com/buggregator/app/blob/master/.rr.yaml#L12
https://roadrunner.dev/docs/http-headers

**Comment by HazyAlex** [comment]
Hello!
I implemented these changes in #73.

Because I'm currently using the latest docker image (v1.8.4), I patched the older (i.e. non-beta) version [here](https://github.com/HazyAlex/app/commit/f5bec67676065170dbf5ef09085ced648adde569).
Would you be willing to accept these changes and release a new version (e.g. v1.8.5) or are you just focusing on the beta and not taking requests for older versions?

**Comment by butschster** [comment]
> Hello! I implemented these changes in #73.
> 
> Because I'm currently using the latest docker image (v1.8.4), I patched the older (i.e. non-beta) version [here](https://github.com/HazyAlex/app/commit/f5bec67676065170dbf5ef09085ced648adde569). Would you be willing to accept these changes and release a new version (e.g. v1.8.5) or are you just focusing on the beta and not taking requests for older versions?

Hi @HazyAlex 

Thank you very much!

**Comment by butschster** [comment]
@HazyAlex I can't release 1.8.5 from master branch. You need to create a new PR to 1.x branch
